src/test_tif_loader.cpp

```diff
@@ -60,14 +60,17 @@ class MapPublisher : public rclcpp::Node {
 
     map_ = std::make_shared<GridMapGeo>();
     map_->Load(file_path, false, color_path);
-    timer_ = this->create_wall_timer(5s, std::bind(&MapPublisher::timer_callback, this));
+    auto timer_callback = [this]() -> void {
+      auto msg = grid_map::GridMapRosConverter::toMessage(map_->getGridMap());
+      if (msg) {
+        msg->header.stamp = now();
+        original_map_pub_->publish(std::move(msg));
+      }
+    };
+    timer_ = this->create_wall_timer(5s, timer_callback);
```

Comment:
Interesting, 


I think for this node it makes a lot of sense since there is not much going on. How much does this apply for general controllers that need to run on a fixed schedule? For me just slightly weird that you would have all the logic implemented in the constructor?